patch-hack.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+# 1. Set the path to the downloaded Hack repo here first
+HACK_PATH=""
+
+# 2. List your alternate glyphs here
+declare -a alternates=(
+    "u00EC-slab"
+    "u00ED-slab"
+    "u00EF-slab"
+    "u012B-slab"
+    "u012D-slab"
+    "u012F-slab"
+    "u0030-diamond"
+    "u0069-slab"
+    "u0129-slab"
+    "u0131-slab"
+    "u0456-slab"
+    "u0457-slab"
+)
+
+# 3. Run this script and enjoy!
+
+# ------------------------------------------------------------------------------------------------ #
+
+if [ -z $HACK_PATH ]; then
+    echo "❌ Set the HACK_PATH variable in this script to your downloaded Hack repo root."
+    exit 1
+fi
+
+declare -a styles=("Regular" "Bold" "Italic" "BoldItalic")
+
+for alternate in "${alternates[@]}"
+do
+    for style in "${styles[@]}"
+    do
+        stylePath="${HACK_PATH}/source/Hack-${style}.ufo/glyphs"
+        lowercaseStyle=`echo "$style" | tr '[:upper:]' '[:lower:]'`
+        alternatePath="./glyphs/${alternate}/${lowercaseStyle}/*.glif"
+
+        cp $alternatePath $stylePath
+        if [ $? -ne 0 ]; then
```

Comment:
from shellcheck: 

```
In patch-hack.sh line 42:
        if [ $? -ne 0 ]; then
             ^-- SC2181: Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.
```

I use the idiom that you used above too.  Thoughts about modifying as suggested here?